src/string/swab.c

```diff
@@ -6,7 +6,11 @@ void swab(const void *restrict _src : itype(restrict _Array_ptr<const void>) byt
 _Checked{
 	_Array_ptr<const char> src : count(n / sizeof(char)) = (_Array_ptr<const char>) _src;
 	_Array_ptr<char> dest: count(n /sizeof(char)) = (_Array_ptr<char>) _dest;
-	for (; n>1; n-=2) {
+
+// n is used in the declared bounds of src and dest. But n is modified in the loop and hence the declared bounds of src and dest no
```

Comment:
Please indent the comment with the code in this function. Also make sure each line of the comment is no longer than 80 chars.